authoring/overview.md

```diff
@@ -99,6 +99,9 @@ pyOpenSci projects should use an open source software license that is approved b
 - **Better/Best:** Choose a license based on your needs and future use of package, plus explain your choice in your submission for review.
 
 ## Other recommendations
+### Python version support
+As keeping compatibility with old Python versions could be a hard task, pyOpenSci suggests that the package should supports, at least, latest three Python versions, as much as possible.
```

Comment:
```suggestion
You should always be explicit about which versions of Python your package supports.
Keeping compatibility with old Python versions can be difficult as functionality changes.
A good rule of thumb is that the package should support, at least,
the latest three Python versions (e.g., 3.8, 3.7, 3.6).
```

Reply:
how about this?

Reply:
sounds great! suggestion applied! thank you so much!